XCUITests/SaveLoginsTests.swift

```diff
@@ -221,4 +206,43 @@ class SaveLoginTest: BaseTestCase {
         let passwordValue = app.webViews.secureTextFields.element(boundBy: 0).value!
         XCTAssertEqual(passwordValue as! String, "••••••••")
     }
+
+    // Smoketest
```

Comment:
@SimonBasca what do you think about this test as part of the smoketest since it is about a new feature?